examples/signatures/src/main.sw

```diff
@@ -0,0 +1,32 @@
+script;
+
+use std::result::Result;
+use std::b512::B512;
+use std::panic::panic;
+use std::chain::log_b256;
+use std::ecr::{
+    ec_recover,
+    ec_recover_address,
+    EcRecoverError
+};
+
+const MSG_HASH = 0xee45573606c96c98ba970ff7cf9511f1b8b25e6bcd52ced30b89df1e4a9c4323;
+
+fn main() {
+    let signature: B512 = ~B512::from(
+        0xbd0c9b8792876713afa8bff383eebf31c43437823ed761cc3600d0016de5110c,
+        0x44ac566bd156b4fc71a4a4cb2655d3dd360c695edb17dc3b64d611e122fea23d
+    );
+
+    // A recovered public key pair.
+    let public_key = ec_recover(signature, MSG_HASH);
+
+    // A recovered Fuel address.
+    let result_address: Result<Address, EcRecoverError> = ec_recover_address(signature, MSG_HASH);
+    if let Result::Ok(address) = result_address {
+        log_b256(address.value);
+    } else {
+        panic(0);
+    };
+}
+
```

Comment:
Format this file. Please turn on format on save in your IDE.

Reply:
Do we have instructions for that somewhere? Can't seem to find any for the Sway formatter particularly.

Reply:
If you're using vscode and have format in save turned on in general, then it should just work automatically.

Reply:
Okay, I think I'd like to notate that in the docs. As it's not immediately clear its available by reading the docs that the plugin is capable of this. But it's really cool!

Reply:
Also, my only hesitation with doing this as well, is that currently the formatting for dependancies / use statements is not pleasant. Until that's resolved, I'll manually run the formatter. But I'll be sure to do though every time.

Reply:
It doesn't really matter that it's not pleasant. There are no guarantees that you or anyone else will _personally_ always find the formatted code pleasant irrespective of this specific case. Please turn on format on save.

Reply:
Until the dependency formatting is resolved, I'll be running it manually.

Formatting all my dependancies in one giant line is not only not pleasant, it's unsafe and should be resolved first.

Reply:
Not exactly sure what's unsafe about that.

Until #1209 is resolved, a workaround would be to split it up into multiple `use` statements.